### ISSUE TITLE
Voiceover doesn't read errors on article deposit form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#2221]
See video.

https://github.com/user-attachments/assets/1307db42-3561-40ae-bae2-39f373358b1a
